app/decorators/event_decorator.rb

```diff
@@ -0,0 +1,24 @@
+class EventDecorator < Draper::Decorator
+  delegate_all
+
+   def self.collection_decorator_class
+    PaginatingDecorator
+  end
+
+  def beginning_at
+    object.beginning_at.strftime("%Y-%m-%d %R")
+  end
+
+  def end_at
+    object.beginning_at.strftime("%Y-%m-%d %R")
+  end
+
+  def short_description
+    max_length = 400
+    if object.description.length > max_length
+      object.description[0, max_length - 3] + "..."
```

Comment:
as far as I can see this method does same thing as build in action view `truncate` method http://apidock.com/rails/ActionView/Helpers/TextHelper/truncate